terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -537,6 +526,50 @@ module "stop_dms_task_job" {
   }
 }
 
+# Glue Job, Activate/Deactivate Glue Trigger Job
```

Comment:
This Glue job allows activating/deactivating a scheduled trigger of another job. Required for deactivating the archiving job when the replay pipeline is in progress.